engine/table/src/main/java/io/deephaven/engine/table/impl/sort/timsort/ByteIntTimsortKernel.java

```diff
@@ -209,24 +231,50 @@ static boolean leq(byte lhs, byte rhs) {
     }
 
     /**
-     * <p>There are two merge invariants that we must preserve, quoting from Wikipedia:</p>
+     * <p>
+     * There are two merge invariants that we must preserve, quoting from Wikipedia:
+     * </p>
      *
-     * <p>Concurrently with the search for runs, the runs are merged with mergesort. Except where Timsort tries to optimise for merging disjoint runs in galloping mode, runs are repeatedly merged two at a time, with the only concerns being to maintain stability and merge balance.</p>
+     * <p>
```

Comment:
I think it would be nice (someday) to edit this so it reflects the current version of the text in the "Merge criteria" section from Wikipedia, and not whatever old version we got this from. The advantages would be: clearer English and no groaners like "Somewhat inappreciably, ..."